docs/dns.md

```diff
@@ -1,51 +1,31 @@
-# DNS Integration with SkyDNS
-Since Kubernetes services changed to assign a single IP address to each service, it is
-now possible to use DNS to resolve a DNS name directly to a Kubernetes service, which 
-would then use Kubernetes' proxy to connect to an appropriate pod running the application 
-pointed to by the service definition.
+# DNS Integration with Kubernetes
+
+As of kubernetes 0.8, DNS is offered as a cluster add-on.  If enabled, a DNS
+Pod and Service will be scheduled on the cluster, and the kubelets will be
+configured to tell individual containers to use the DNS Service's IP.
+
+Every Service defined in the cluster (including the DNS server itself) will be
+assigned a DNS name.  By default, a client Pod's DNS search list will
+include the Pod's own namespace and the cluster's default domain.  This is best
+illustrated by example.
+
+Assume a Service named `foo` in namespace `bar`.  A Pod running in namespace
```

Comment:
Be completely explicit here and say "Kubernetes namespace", to disambiguate from DNS namespace.  They happen to match, but the reader doesn't know that yet.